package.json

```diff
@@ -1,14 +1,14 @@
 {
   "scripts": {
-    "start": "APP_BUILD_VERSION=1 APP_BINARY_VERSION=dev START_STORYBOOK=false expo start -c --dev-client",
-    "start:mock": "APP_BUILD_VERSION=1 APP_BINARY_VERSION=dev MOCK=true expo start",
+    "start": "APP_BUILD_VERSION=1 APP_BINARY_VERSION=dev START_STORYBOOK=false npx expo start -c --dev-client",
```

Comment:
just curious why was `npx` added here ah? since expo is already installed, should work without `npx`?

Reply:
`npx` allows you to use the expo installed in this node modules if exists. Else it'll search on the global space.

If don't use `npx` then it'll always try to search your global space. 

- the old non-npx way (global) has been deprecated: https://docs.expo.dev/archive/expo-cli/
- https://docs.expo.dev/more/expo-cli/ <- modern current version